clientGQL.go

```diff
@@ -3,10 +3,11 @@ package opslevel
 import (
 	"context"
 	"fmt"
-	"github.com/hashicorp/go-retryablehttp"
-	"github.com/hasura/go-graphql-client"
 	"net/http"
 	"strings"
+
+	"github.com/hashicorp/go-retryablehttp"
+	"github.com/hasura/go-graphql-client"
```

Comment:
gofumpt changes that fail go vet

Reply:
The `test` github action related to this includes running `task lint` defined in the [Taskfile](https://github.com/OpsLevel/opslevel-go/blob/main/Taskfile.yml#L16). Specifically it runs `gofumpt -d .` with the `-d` flag printing a diff - gofumpt itself doesn't write any changes.

The failure is from `go vet ./...` 
```
Error: vet: ./infra_test.go:288:3: too few values in struct literal of type struct{Request string; Variables string; Response string}
```
caused by some changes made in a [recently merge PR](https://github.com/OpsLevel/opslevel-go/pull/264/files#diff-7fe36a091d93790fb1a943d0636ab952e254e8d8b090881cd19d57e3d4d503ebR70-R72)